scripts/pull-templates.py

```diff
@@ -14,7 +14,7 @@ def parse_options(args=None):
     parser = OptionParser(args)
     parser.add_option("-s", "--server", dest="server", default="http://openlibrary.org/", help="URL of the openlibrary website (default: %default)")
     parser.add_option("--template-root", dest="template_root", default="/upstream", help="Template root (default: %default)")
-    parser.add_option("--default-plugin", dest="default_plugin", default="upstream", help="Default plugin (default: %default)")
+    parser.add_option("--default-plugin", dest="default_plugin", default="/upstream", help="Default plugin (default: %default)")
```

Comment:
Once a line is edited, that line must pass a flake8 test that limits line length to 88 characters max. so please wrap this line so that the Travis tests go green.
```
scripts/pull-templates.py:17:89: E501 line too long (128 > 88 characters)
1     E501 line too long (128 > 88 characters)
```